bridges/dw_hdmi/aw_de2_dw_hdmi.c

```diff
@@ -665,6 +922,34 @@ dw_hdmi_write(struct dw_hdmi_softc *sc, uint32_t reg, uint32_t val)
 	}
 }
 
+static int
+dw_hdmi_register_ports(device_t dev)
+{
+	phandle_t ports, port;
+	phandle_t node, child;
+	char endp[DW_HDMI_MAX_PORTS];
+	int i;
+
+	node = ofw_bus_get_node(dev);
+
+	ports = ofw_bus_find_child(node, "ports");
```

Comment:
This should use the ofw_graph functions.

Reply:
Thanks. I removed this chunk as it was not needed